package.json

```diff
@@ -28,124 +28,124 @@
+    "@opentelemetry/instrumentation-pino": "^0.34.5",
+    "@opentelemetry/resource-detector-container": "^0.3.5",
+    "@opentelemetry/resource-detector-gcp": "^0.29.5",
+    "@opentelemetry/sdk-node": "^0.47.0",
+    "@sendgrid/eventwebhook": "^8.0.0",
     "@sendgrid/mail": "^8.1.0",
-    "@slack/webhook": "^7.0.1",
+    "@slack/webhook": "^7.0.2",
     "@types/shortid": "0.0.32",
     "apollo-server-errors": "^3.3.0",
     "apollo-server-types": "^3.8.0",
-    "cloudinary": "^1.41.0",
-    "csv-parse": "^5.5.2",
+    "cloudinary": "^1.41.3",
+    "csv-parse": "^5.5.3",
     "dataloader": "^2.2.2",
     "date-fns": "^2.30.0",
     "date-fns-tz": "^2.0.0",
     "eventsource": "^2.0.2",
-    "fast-json-stringify": "^5.9.1",
-    "fastify": "^4.24.3",
+    "fast-json-stringify": "^5.10.0",
+    "fastify": "^4.25.2",
     "fastify-plugin": "^4.5.1",
-    "fastify-raw-body": "^4.2.2",
-    "fastq": "^1.15.0",
+    "fastify-raw-body": "^4.3.0",
+    "fastq": "^1.16.0",
     "form-data": "^4.0.0",
     "graphql": "^16.8.1",
     "graphql-parse-resolve-info": "^4.13.0",
-    "graphql-rate-limit-directive": "^2.0.4",
+    "graphql-rate-limit-directive": "^2.0.5",
     "graphql-relay": "^0.10.0",
     "graphql-scalars": "^1.22.4",
     "graphql-subscriptions": "^2.0.0",
-    "graphql-tools": "^8.3.20",
     "graphql-upload": "^15.0.1",
     "he": "^1.2.0",
     "highlight.js": "^11.9.0",
     "ioredis": "^5.3.2",
-    "isbot": "^3.7.1",
+    "isbot": "^4.4.0",
     "jsonexport": "^3.2.0",
     "jsonwebtoken": "^9.0.2",
     "lodash": "^4.17.21",
-    "markdown-it": "^13.0.2",
+    "markdown-it": "^14.0.0",
     "markdown-to-txt": "^2.0.1",
-    "mercurius": "^13.2.2",
-    "mercurius-cache": "^5.0.0",
+    "mercurius": "^13.3.3",
+    "mercurius-cache": "^6.0.1",
     "mercurius-upload": "^6.0.1",
     "nanoid": "^3.3.4",
     "node-fetch": "^2.7.0",
-    "node-html-parser": "^6.1.11",
+    "node-html-parser": "^6.1.12",
     "opentelemetry-instrumentation-typeorm": "^0.40.0",
     "p-timeout": "4.1.0",
     "pg": "^8.11.3",
     "pg-query-stream": "^4.5.3",
-    "pino": "^8.16.1",
-    "rate-limiter-flexible": "^3.0.4",
-    "reflect-metadata": "^0.1.13",
+    "pino": "^8.17.2",
+    "rate-limiter-flexible": "^4.0.0",
+    "reflect-metadata": "^0.2.1",
     "retry": "^0.13.1",
     "rss": "^1.2.2",
-    "typeorm": "0.3.17",
+    "typeorm": "0.3.19",
     "ua-parser-js": "^1.0.37",
     "uuid": "^9.0.1",
     "validate.js": "^0.13.1"
   },
   "devDependencies": {
-    "@faker-js/faker": "^8.1.0",
+    "@faker-js/faker": "^8.3.1",
     "@fastify/static": "^6.12.0",
-    "@swc/core": "^1.3.100",
+    "@swc/core": "^1.3.104",
     "@types/express": "^4.17.21",
-    "@types/jest": "^29.5.8",
+    "@types/jest": "^29.5.11",
     "@types/jsonwebtoken": "^9.0.5",
-    "@types/lodash": "^4.14.201",
+    "@types/lodash": "^4.14.202",
     "@types/node": "^18.16.17",
-    "@types/node-fetch": "^2.6.9",
+    "@types/node-fetch": "^2.6.11",
```

Comment:
I don't think we really need node-fetch anymore since Node 18.x has native fetch, one thing to check are the workers, but I think they run on the same version.

Reply:
Hmm agreed, but i'll leave it as seperate from the bumps to avoid complications